### ISSUE TITLE
Build error on Fedora 32

### ISSUE THREAD

**Issue by marmulak** [masari-project/masari#187]
fedora-release: Fedora release 32 (Thirty Two)
gcc version 10.1.1 20200507 (Red Hat 10.1.1-1) (GCC)

[make.log](https://github.com/masari-project/masari/files/4711858/make.log)

**Comment by TheGreatMcPain** [comment]
I know it's almost been a year, but were you able to get it built?

I've been working on an ebuild for Gentoo which now builds, but I had to add a bunch of patches, because of boost shuffling things around.

**Comment by marmulak** [comment]
No, it has never built. I'm on Fedora 33 now, here is the build error just now:

[errors.txt](https://github.com/masari-project/masari/files/6002905/errors.txt)

**Comment by TheGreatMcPain** [comment]
What version of boost do you have installed?

I'm currently on 1.74.0

**Comment by marmulak** [comment]
boost-1.73.0-9.fc33.x86_64

**Comment by TheGreatMcPain** [comment]
It might be a good idea to try [#188](https://github.com/masari-project/masari/pull/188) again.
